solr/core/src/test/org/apache/solr/TestDistributedGrouping.java

```diff
@@ -29,8 +29,11 @@
 import org.apache.solr.common.params.ModifiableSolrParams;
 import org.apache.solr.common.util.NamedList;
 import org.apache.solr.SolrTestCaseJ4.SuppressPointFields;
+import org.junit.Assert;
```

Comment:
I think we don't need this import. All test cases extend `Assert`

Reply:
I think `Assert` is still needed  (I use `Assert.assertThat`, code won't compile otherwise).

Reply:
What I meant was since all tests extend Assert.. you could just use assertThat instead of Assert.assertThat

Reply:
Ouch sorry, I didn't understand, done! thanks!